sklearn/neighbors/quad_tree.pxd

```diff
@@ -0,0 +1,88 @@
+# cython: boundscheck=False
```

Comment:
since this file is not used in any neighbors code, I find it weird to have in this subpackage. why not utils?